src/app/table/table.component.ts

```diff
@@ -139,4 +140,15 @@ export class GpfTableComponent implements OnChanges, AfterViewChecked {
     const scrollIndices = this.getScrollIndices();
     return this.dataSource.slice(scrollIndices[0], scrollIndices[1]);
   }
+
+  public calculateTableWidthFloat(): number {
+    let tableWidthFloat = 0;
+    if (this.columnsChildren !== undefined && this.columnsChildren !== null) {
```

Comment:
Invert this condition so that it works as a return guard, could also use `if (!this.columnsChildren)`